rocketpy/stochastic/stochastic_environment.py

```diff
@@ -52,8 +43,8 @@ def __init__(
         latitude=None,
         longitude=None,
         ensemble_member=None,
-        wind_velocity_x_factor=(1, 0),
-        wind_velocity_y_factor=(1, 0),
+        wind_velocity_x_factor=(1, 0),  # TODO: please remove it
+        wind_velocity_y_factor=(1, 0),  # TODO: please remove it
     ):
         """Initializes the Stochastic Environment class.
```

Comment:
(Just want to let it clear before I start: I'm open mind and can change the opinion below as discussion.)

Having these factors for drag coeeficients make a lot of sense but I think it is not **correct** if we multiply the whole wind profile by the same constant number. Statistically, it is not sophisticated and I personally cannot guarantee it is representative enough. The wind speed is not the same for all the pressure levels, and therefore the variance of this wind speed cannot be the same at all the pressure levels.
I know that in the past I suggested adding these factors here, but have to be more careful now. I'm not so confortable with the multiplier factors for the whole wind profile, especially given that you managed to vary the ensemble members during the simulation, which is sophisticated enough as the ensemble models are based on validated statistical model.

However... There is the **particular** case where we define the wind speed as a constant value. In this case, it more a lot of sense to me that we apply a multiplying factor. Nevertheless, it would still be possible to vary this constant value using the standard format we are using for the stochastic models: (mean, std, "distribution").

Summary points:
- These wind factor seems to only make sense for a simple case where you assign a constant value to the wind speed.
- The Ensemble member variance is a more "statistically sustained" approach, but it requires the user to set an ensemble. 
- Varying the ensemble member and the wind_factor simultainsly - which is the case our the current implementation - is not good. 

One thing that I think we should consider: who is our target audience for these Monte Carlo simulations?

So my final question now: @MateusStano do you still thikn keeping these wind factor is a good idea?

I would love to have third opinion on this specificy topic (@phmbressan @Lucas-Prates)

Reply:
I think we should clearly state in the wind factor docs that it should not be used with height-varying wind profiles

Reply:
I agree with Stano here, I believe there is use in keeping the wind factor, but state that it might be non-physical in height-varying wind profiles.

Reply:
Done!